lib/engine/game/g_18_esp/step/choose.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G18ESP
+      module Step
+        class Choose < Engine::Step::Base
+          def description
+            'Choose a Train'
+          end
+
+          def actions(entity)
+            actions = []
+            actions << 'choose' if @game.p2&.owner == entity
```

Comment:
Should this skip be stepped if the corporation is at train limit?

Reply:
according to the rules, when acquiring the company closes. The train remains.
If bought while train limited, the company closes, and you need to dump a train (either this one, or some other one)

Reply:
If i added a check here on train limit, the result would be company is still open and pays out to the owning corp, which is wrong

Reply:
waiting to hear from the game dev as to what happenes when p2 is bought while the owning corp is train locked

Reply:
When buying the company by and owning corp that is train limited, it gains the 2 train, but then needs to discard some other train. Therefore no need to check train spot when buying/choosing train